### PR TITLE
LIE/HM part 1: prerequisite

### PR DESCRIPTION
**Pull request by norihiro-w**
a part of #1535

This PR includes changes required for implementing HM process with the LIE approach. Please check each commit

### CODE REVIEW

File: MaterialLib/FractureModels/FractureIdentity2.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+#include <Eigen/Eigen>
+
+namespace MaterialLib
+{
+namespace Fracture
+{
+
+template <int DisplacementDim>
+struct FractureIdentity2
```
Comment (chleh) [inline MaterialLib/FractureModels/FractureIdentity2.h]: Could you please add some documentation? And why is there a two at the end of the class name? At least I cannot see an already existing file `FractureIdentity.h`?

**Comment by chleh** [comment]
:+1:

File: MaterialLib/FractureModels/FractureIdentity2.h (hunk 1 above)
Comment (norihiro-w) [inline MaterialLib/FractureModels/FractureIdentity2.h]: this corresponds to `Invariants::identity2` in KelvinVector.h for usual solids. I will add some comment there

**Comment by norihiro-w** [comment]
Something is going wrong with conan and that's why Travis is failing. this is nothing to do with this PR.

**Comment by norihiro-w** [comment]
@wenqing or @endJunction can you please review this?

**Pull request merged**